mpp/mpp_domains.F90

```diff
@@ -161,6 +161,7 @@ module mpp_domains_mod
   public :: mpp_clear_group_update
   public :: mpp_group_update_initialized, mpp_group_update_is_set
   public :: mpp_get_global_domains
+  public :: mpp_create_super_grid_domain
```

Comment:
Since this is a destructive operation, there should be doxygen information on how to utilize this function (copy, modify).